src/core/Akka/Actor/BuiltInActors.cs

```diff
@@ -15,7 +15,7 @@ namespace Akka.Actor
     /// <summary>
     ///     Class EventStreamActor.
     /// </summary>
-    public class EventStreamActor : ActorBase
+    /*TODO: this class is not used*/public class EventStreamActor : ActorBase
```

Comment:
Anyone? Just an unfinished impl? Is it a leftover from the early days? 
Im feeling guilty but dont know if I did this :)